### PR TITLE
Implement carousel

### PR DESCRIPTION
**Pull request by kart-c**
## Implement carousel feature on home page

- Earlier only `i` tags were provided which would navigate through images when clicked.
- Wrapped `i` tags in the `button` tag for accessibility purposes.
- [CSB for reference](https://8jjl6c.csb.app/)

caarousel

### CODE REVIEW

File: src/Pages/Home/Components/Carousel/Carousel.jsx [hunk 1]
```diff
@@ -1,16 +1,31 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
+import { carouselImgs } from '../../../../Assets/images';
 import styles from './Carousel.module.css';
-import { Carousel1 } from '../../../../Assets/images';
 
 const Carousel = () => {
+	const [currentImg, setCurrentImg] = useState(0);
```
Comment (AnkurChunekar) [inline src/Pages/Home/Components/Carousel/Carousel.jsx]: For more clarity and readability you can name `currentImage` as `currentImgIndex`.

**Review by AnkurChunekar — APPROVED** [review]
Rest everything looks great, code is clean and well indented.

File: src/Pages/Home/Components/Carousel/Carousel.jsx (hunk 1 above)
Comment (kart-c) [reply src/Pages/Home/Components/Carousel/Carousel.jsx]: I was going to change that but forgot. Thank you, Ankur for noticing.

**Review by AtulPant2704 — APPROVED** [review]
Code is clean and easy to understand, Carousel also looks really good.

**Pull request merged**